errors.go

```diff
@@ -238,6 +245,12 @@ func HandleHTTPError(ctx context.Context, err error) error {
 		errResponse.Detail = errorDetail.DetailMsg()
 	}
 
+	// Check for title
+	var errorTitle ErrorWithTitle
+	if errors.As(err, &errorTitle) {
+		errResponse.Title = errorTitle.TitleString()
+	}
```

Comment:
I feel you are replicating the pattern that exists above, but I would prefer to apply the logic of error variable always starts with err


```suggestion
	// Check for title
	var errWithTitle ErrorWithTitle
	if errors.As(err, &errWithTitle) {
		errResponse.Title = errWithTitle.TitleString()
	}
```

Reply:
Updated with co author'd